src/pages/RoomMembersPage.tsx

```diff
@@ -92,7 +92,7 @@ function RoomMembersPage({report, policy}: RoomMembersPageProps) {
     // eslint-disable-next-line rulesdir/prefer-shouldUseNarrowLayout-instead-of-isSmallScreenWidth
     const {shouldUseNarrowLayout, isSmallScreenWidth} = useResponsiveLayout();
     const [selectionMode] = useOnyx(ONYXKEYS.MOBILE_SELECTION_MODE, {canBeMissing: true});
-    const canSelectMultiple = isSmallScreenWidth ? selectionMode?.isEnabled : true;
+    const canSelectMultiple = isSmallScreenWidth ? !!selectionMode : true;
```

Comment:
is there any reason that this component uses `useOnyx` directly? See line 94. I think we can try to use the hook here